gitlab/v4/cli.py

```diff
@@ -207,12 +207,20 @@ def _populate_sub_parser_by_class(
     mgr_cls = getattr(gitlab.v4.objects, mgr_cls_name)
 
     action_parsers: Dict[str, argparse.ArgumentParser] = {}
-    for action_name in ["list", "get", "create", "update", "delete"]:
+    for action_name, help_text in [
+        ("list", "List the GitLab resources"),
+        ("get", "Get a GitLab resource"),
+        ("create", "Create a GitLab resource"),
+        ("update", "Update a GitLab resource"),
+        ("delete", "Delete a GitLab resource"),
```

Comment:
This might have some additional meaning in a few cases (revoke/deactivate etc) but can't think of a way to document that here, LGTM :+1: